src/components/pages/Dashboard.vue

```diff
@@ -516,10 +528,10 @@ export default {
         // Check that there are valid trials
         var trials = obj['trials'];
         // Filter trials by name and status.
-        trials = trials.filter(trial => trial.status === 'done' && trial.name !== 'neutral' && trial.name !== 'calibration')
+        trials = trials.filter(trial => trial.status === 'done'&& trial.name !== 'calibration')
 
         if (trials.length > 0) {
-          return obj.meta.subject.id + " (" + obj.id + ")";
+          return  obj.name + " (" + obj.id + ")";
```

Comment:
Let's make sure it is backward compatible.

Reply:
I added an if-else case in the latest commit